File: BOJ/5000 ~ 6000/P5622.java
import java.util.Map;
import java.util.Scanner;

public class P5622 {
    public static void main(String[] args) {
        Map<Character, Integer> dial = Map.ofEntries(
                Map.entry('A', 2),
                Map.entry('B', 2),
                Map.entry('C', 2),
                Map.entry('D', 3),
                Map.entry('E', 3),
                Map.entry('F', 3),
                Map.entry('G', 4),
                Map.entry('H', 4),
                Map.entry('I', 4),
                Map.entry('J', 5),
                Map.entry('K', 5),
                Map.entry('L', 5),
                Map.entry('M', 6),
                Map.entry('N', 6),
                Map.entry('O', 6),
                Map.entry('P', 7),
                Map.entry('Q', 7),
                Map.entry('R', 7),
                Map.entry('S', 7),
                Map.entry('T', 8),
                Map.entry('U', 8),
                Map.entry('V', 8),
                Map.entry('W', 9),
                Map.entry('X', 9),
                Map.entry('Y', 9),
                Map.entry('Z', 9)
        );

        Scanner sc = new Scanner(System.in);
        String word = sc.next();
        int result = 0;
        for (char c: word.toCharArray()) {
            Integer dialNumber = dial.get(c);
            result += (dialNumber + 1);
        }
        System.out.println(result);
    }
}

File: BOJ/5000 ~ 6000/P5622.kt
// https://www.acmicpc.net/problem/5622

fun main() {
    val word = readln()
    var result = 0

    for(c in word.toCharArray()) {
        when(c) {
            'A', 'B', 'C' -> result += 3
            'D', 'E', 'F' -> result += 4
            'G', 'H', 'I' -> result += 5
            'J', 'K', 'L' -> result += 6
            'M', 'N', 'O' -> result += 7
            'P', 'Q', 'R', 'S' -> result += 8
            'T', 'U', 'V' -> result += 9
            'W', 'X', 'Y', 'Z', -> result += 10
        }
    }
    println(result)
}
